lib/windows_utils.pm

```diff
@@ -24,15 +24,14 @@ sub wait_boot_windows {
     my %args            = @_;
     my $bootloader_time = $args{bootloader_time} // 100;
 
-    assert_screen 'windows-bootsplash', 600;
+    assert_screen 'windows-screensaver', 600;
 
     # Reset the consoles after the reboot: there is no user logged in anywhere
     reset_consoles;
 
-    assert_screen 'windows-screensaver', 600;
-
     send_key 'esc';    # press shutdown button
 
+    assert_screen 'windows-network';
```

Comment:
Yes, the two asserts (network/login) have same screen but differenet region to compare. 
Since the network may not be ready in the earlier step http://147.2.212.93/tests/616#step/boot_windows/2, so I add this assert just want to explicitly check the network status before doing the login operation.

I agree with have one needle to assert network and login at the same time.